### PR TITLE
Update CLI version, remove JSON serializer APIs and rename Encodings to TextEncodings to avoid conflict.

### PR DESCRIPTION
**Pull request by ahsonkhan**
This helps unblock @Gnbrkm41: https://github.com/dotnet/corefxlab/pull/2673#issuecomment-491459711

cc @steveharter, @GrabYourPitchforks

### CODE REVIEW

File: tests/System.Text.JsonLab.Tests/JsonObjectTests.cs [hunk 1]
```diff
@@ -212,8 +212,9 @@ public void ParseJson(bool compactData, TestCaseType type, string jsonString)
 
                 if (type == TestCaseType.Json400KB)
                 {
-                    expectedString = ReadJson400KB(jtoken);
-                    actualString = ReadJson400KB(obj);
+                    // TODO: Investigate why 1 + 75486/(double)100000 == 1.7548599999999999 instead of just 1.75846
```
Comment (ahsonkhan) [inline tests/System.Text.JsonLab.Tests/JsonObjectTests.cs]: @tannergooding, is this expected behavior? Shows up in 3.0 only (not in netfx/.NET Core 2.1):
```C#
double x = 1 + 75486/(double)100000;
Console.WriteLine(x);   // 1.7548599999999999
```

**Comment by ahsonkhan** [comment]
@safern, the CI isn't running all the legs. Do you know why?

**Comment by safern** [comment]
> @safern, the CI isn't running all the legs. Do you know why?

I know there is an Azure DevOps bug with the github webhooks we have a thread going on. Don’t know if this is what is hitting but looking now.

**Comment by safern** [comment]
The build is running but for some reason GH it is not showing it:

https://dev.azure.com/dnceng/public/_build/results?buildId=188138

**Comment by ahsonkhan** [comment]
Since the build was actually successful, merging to unblock other contributors (https://dev.azure.com/dnceng/public/_build/results?buildId=188146). Will keep an eye out for any official build failures, just in case.

File: tests/System.Text.JsonLab.Tests/JsonObjectTests.cs (hunk 1 above)
Comment (tannergooding) [reply tests/System.Text.JsonLab.Tests/JsonObjectTests.cs]: Yes. The raw bits for `1 + 75486 / (double)100000` are `0x3FFC13E81450EFDC`
The raw bits for `1.75486` are `0x3FFC13E81450EFDD`

In full framework, `ToString()` returned `1.75486` for both of them, which meant you couldn't differentiate.

In core, we return the shortest roundtrippable string, which allows them to always be differentiated.

**Pull request merged**